step1-03/demo/index.html

```diff
@@ -1,15 +1,9 @@
 <html>
-  <body>
-    <div
-      class="codepen"
-      data-theme-id="36294"
-      data-prefill
-      data-editable="true"
-      data-height="100%"
-      data-theme-id="1"
-      data-default-tab="js,result"
-    >
-      <pre data-lang="css">
+
+<body>
```

Comment:
Would you please revert all the changes to this file aside from adding the `for` attribute to the label?

Reply:
Done.